docs/choosing_clusters.md

```diff
@@ -220,7 +220,46 @@ Here's an example:
 
 A local registry is often the fastest way to speed up your dev experience.
 
-Every cluster sets up this registry slightly differently. 
+Every cluster sets up this registry slightly differently.
+
+Tilt-team is currently collaborating with the Kubernetes community on how
+protocols for discovery, so that local development tools will auto-configure
+when a local registry is present.
+
+Tilt currently supports two generic protocols. We've documented both below.
+The Kubernetes standard protocol is a KEP that has been vetted by the Kubernetes community.
+The annotation-based protocol is used in legacy Tilt scripts.
+
+#### Kubernetes Standard Registry Discovery
+
+The new protocol uses configmaps in the `kube-public` namespace of your cluster.
```

Comment:
what new protocol? either "This protocol uses..." or "This is a new protocol that uses..."

Reply:
done